tenants-certified-attributes-updater/src/main/scala/it/pagopa/interop/tenantscertifiedattributesupdater/repository/impl/TenantRepositoryImpl.scala

```diff
@@ -16,6 +17,6 @@ final case class TenantRepositoryImpl(client: MongoClient) extends TenantReposit
       .getCollection(ApplicationConfiguration.tenantsCollection)
 
   def getTenants: Future[Seq[Either[Throwable, PersistentTenant]]] =
-    collection.find().map(extractData[PersistentTenant]).toFuture()
+    collection.find(Filters.exists("data.selfcareId")).map(extractData[PersistentTenant]).toFuture()
```

Comment:
`exists` returns also records with `selfcareId = null`.
In order to filter also null values we need to use 
```suggestion
    collection.find(Filters.ne("data.selfcareId", null)).map(extractData[PersistentTenant]).toFuture()
```